doc/architecture/adr/0022-telemetry.md

```diff
@@ -0,0 +1,30 @@
+# Telemetry
+
+* Status: Accepted
+* Deciders: @MortMH, @CodeReaper, @duizer, @ckr123
+* Date: 2023-05-09
+
+---
+
+## Context and Problem Statement
+
+With the increasing complexity of modern distributed systems, it's becoming more and more difficult to trace requests across services, identify performance bottlenecks, and diagnose issues. Traditional monitoring and logging approaches can only provide limited visibility into the system, leaving blind spots and making it hard to pinpoint problems. To address this issue, developers and DevOps teams need a standardized and extensible approach for capturing and analyzing telemetry data across their entire stack.
+
+---
+
+## Considered Options
+
+* Direct integration with Prometheus/Jaeger
+* OpenTelemetry
```

Comment:
Telemetry is often broken down into logs, metrics and traces (this applies to OpenTelemetry at least). Reading this ADR it seems to be ignoring the logs part. Have you/we given any thoughts on logs and OpenTelemetry?